Comp2023-Imported/src/main/java/frc/robot/Constants.java

```diff
@@ -512,19 +512,13 @@ public enum VIRequests
     ));
 
     // Direction of goal relative to AprilTag 
-    public enum VIGoalDirection
-    {
-      DIRECTION_LEFT,   // Left
-      DIRECTION_MIDDLE, // Middle
-      DIRECTION_RIGHT   // Right
-    }
 
-    public static final double kATagDepthInGrid    = Units.inchesToMeters(14.25);    // Depth from front of grid to AprilTag - 1'2-1/4"
-    public static final double kRobotCenterToFront = Units.inchesToMeters((28.0 + 6.0) / 2); // Depth from limelight to front robot edge
-    public static final double kAdjustPathX        = kATagDepthInGrid + kRobotCenterToFront;
-    public static final double kAdjustPathY        = Units.inchesToMeters(18.25 / 2 + 18.5 / 2) + 0.06;     // Addition of 6cm to adjust for empirical error 
-    public static final double kAdjustSubPathX     = kRobotCenterToFront + Units.inchesToMeters(30); // Robot stop 30 inches from the substation loading zone
-    public static final double kAdjustSubPathY     = Units.inchesToMeters(50.5 / 2);
+    public static final double       kATagDepthInGrid    = Units.inchesToMeters(14.25);    // Depth from front of grid to AprilTag - 1'2-1/4"
```

Comment:
These are all geometry for last year's game. Note the reference to the grid. And the path adjustments were to the nodes with respect to the center of the robot.